bundle/src/main/java/com/adobe/acs/commons/redirects/servlets/ExportRedirectMapServlet.java

```diff
@@ -0,0 +1,137 @@
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.adobe.acs.commons.redirects.servlets;
+
+import com.adobe.acs.commons.redirects.filter.RedirectFilter;
+import com.adobe.acs.commons.redirects.models.RedirectRule;
+import org.apache.poi.ss.usermodel.Cell;
+import org.apache.poi.ss.usermodel.CellStyle;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Font;
+import org.apache.poi.ss.usermodel.IndexedColors;
+import org.apache.poi.ss.usermodel.FillPatternType;
+import org.apache.poi.ss.util.CellRangeAddress;
+import org.apache.poi.xssf.usermodel.XSSFCellStyle;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.apache.sling.api.SlingHttpServletRequest;
+import org.apache.sling.api.SlingHttpServletResponse;
+import org.apache.sling.api.resource.Resource;
+import org.apache.sling.api.servlets.SlingSafeMethodsServlet;
+import org.osgi.service.component.annotations.Component;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.Servlet;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.Collection;
+import java.util.Date;
+
+import static com.adobe.acs.commons.redirects.filter.RedirectFilter.ACS_REDIRECTS_RESOURCE_TYPE;
+
+
+/**
+ * A servlet to export redirect configurations into an Excel spreadsheet
+ *
+ */
+@Component(service = Servlet.class, immediate = true, name = "ExportRedirectMapServlet", property = {
+        "sling.servlet.label=ACS AEM Commons - Export Redirects Servlet",
+        "sling.servlet.methods=GET",
+        "sling.servlet.selectors=export",
+        "sling.servlet.resourceTypes=" + ACS_REDIRECTS_RESOURCE_TYPE
+})
+public class ExportRedirectMapServlet extends SlingSafeMethodsServlet {
+
+    private static final Logger log = LoggerFactory.getLogger(ExportRedirectMapServlet.class);
+    private static final long serialVersionUID = -3564475196678277711L;
+
+    static final String SPREADSHEETML_SHEET = "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet";
+
+    @Override
+    protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse response)
+            throws IOException {
+
+        String path = request.getParameter("path");
+        Resource root = request.getResourceResolver().getResource(path);
+        log.debug("Requesting redirect maps from {}", path);
+
+        Collection<RedirectRule> rules = RedirectFilter.getRules(root);
+        XSSFWorkbook wb = export(rules);
+
+        response.setContentType(SPREADSHEETML_SHEET);
+        response.setHeader("Content-Disposition", "attachment;filename=\"acs-redirects.xlsx\" ");
+        wb.write(response.getOutputStream());
+    }
+
+    static XSSFWorkbook export(Collection<RedirectRule> rules) {
+        XSSFWorkbook wb = new XSSFWorkbook();
```

Comment:
@YegorKozlov wow! it's like your super familiar with apache POI or something! ;)

Been a long time - hope you've been well!

Reply:
Hey @davidjgonzalez 

>  it's like your super familiar with apache POI or something! ;)

Yep, wearing my Apache POI committer hat on :)